app/models/test_track/lazy_visitor_by_identity.rb

```diff
@@ -34,5 +34,7 @@ def __load_visitor__
       id: remote_visitor.id,
       assignments: remote_visitor.assignments
     )
+  rescue *TestTrack::SERVER_ERRORS => e
+    raise TestTrackRailsClient::UnrecoverableConnectivityError, e
```

Comment:
Thinking on using the TestTrackRailsClient module instead of TestTrack? I believe we decided to go ahead and use TestTrack namespace for the whole client just to keep it short and to the point, because you'll never be running the server code here, so it shouldn't be confusing.